src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/SystemCommandLine/Utils.tt

```diff
@@ -211,6 +211,10 @@ public class OptionData
             default:
                 throw new Exception("Unknown Element Type " + element.Name.LocalName);
         }
+
+        var textInfo = CultureInfo.InvariantCulture.TextInfo;
+        CamelCaseName = $"{textInfo.ToLower(InitCapsName[0])}{InitCapsName.Substring(1)}";
```

Comment:
Is it guaranteed that InitCapsName will be non-empty? If not, the Substring will fail.